zest/releaser/prerelease.py

```diff
@@ -127,6 +132,24 @@ def _grab_history(self):
         # TODO: add line number where an extra changelog entry can be
         # inserted.
 
+        # Look for 'Nothing changed yet' under the latest header.  Not
+        # nice if this text ends up in the changelog.  Did nothing happen?
+        start = headings[0]['line']
```

Comment:
If headings is empty, we have already raised an error in the condition around line 110 ("No detectable version heading in the history file").